linux_os/guide/system/software/integrity/software-integrity/aide/aide_scan_notification/bash/shared.sh

```diff
@@ -3,9 +3,28 @@
 {{{ bash_package_install("aide") }}}
 {{{ bash_instantiate_variables("var_aide_scan_notification_email") }}}
 
-CRONTAB=/etc/crontab
-CRONDIRS='/etc/cron.d /etc/cron.daily /etc/cron.weekly /etc/cron.monthly'
-
```

Comment:
@teacup-on-rockingchair  I think this shouldn't be removed, these variables are used below (lines 37-38). I think this might be the reason why this rule fails the CI.